services/wallet/router/router_v2.go

```diff
@@ -266,12 +266,6 @@ func findBestV2(routes [][]*PathV2, tokenPrice float64, nativeChainTokenPrice fl
 	for _, route := range routes {
 		currentCost := big.NewFloat(0)
 		for _, path := range route {
-			if path.FromToken.IsNative() {
-				path.requiredNativeBalance = new(big.Int).Set(path.AmountIn.ToInt())
-			} else {
-				path.requiredTokenBalance = new(big.Int).Set(path.AmountIn.ToInt())
-				path.requiredNativeBalance = big.NewInt(0)
-			}
```

Comment:
why removing this logic?

Reply:
I didn't want to remove it, just to remove a condition, cause I need this:
```
path.requiredTokenBalance = new(big.Int).Set(path.AmountIn.ToInt())
path.requiredNativeBalance = big.NewInt(0)
```
in both cases, but when I was cleaning the code I missed that and you were very fast in reviewing. :)